src/winkerberos.c

```diff
@@ -878,18 +1044,28 @@ sspi_client_wrap(PyObject* self, PyObject* args) {
 static PyMethodDef WinKerberosClientMethods[] = {
     {"authGSSClientInit", (PyCFunction)sspi_client_init,
      METH_VARARGS | METH_KEYWORDS, sspi_client_init_doc},
+	{ "authGSSServerInit", (PyCFunction)sspi_server_init,
+	 METH_VARARGS | METH_KEYWORDS },
     {"authGSSClientClean", sspi_client_clean,
      METH_VARARGS, sspi_client_clean_doc},
+	{ "authGSSServerClean", sspi_server_clean,
+	 METH_VARARGS },
     {"channelBindings", (PyCFunction)sspi_channel_bindings,
      METH_VARARGS | METH_KEYWORDS, sspi_channel_bindings_doc},
     {"authGSSClientStep", (PyCFunction)sspi_client_step,
      METH_VARARGS | METH_KEYWORDS, sspi_client_step_doc},
+	{ "authGSSServerStep", (PyCFunction)sspi_server_step,
+	 METH_VARARGS | METH_KEYWORDS },
     {"authGSSClientResponse", sspi_client_response,
      METH_VARARGS, sspi_client_response_doc},
+	{ "authGSSServerResponse", sspi_server_response,
+	 METH_VARARGS },
     {"authGSSClientResponseConf", sspi_client_response_conf,
      METH_VARARGS, sspi_client_response_conf_doc},
     {"authGSSClientUsername", sspi_client_username,
      METH_VARARGS, sspi_client_username_doc},
+	{ "authGSSServerUsername", sspi_server_username,
```

Comment:
It was pointed out in https://github.com/mongodb-labs/winkerberos/issues/25 that the function name on the client side should be authGSSClientUserName (Name is capitalized). This should be authGSSServerUserName. See https://github.com/apple/ccs-pykerberos/blob/PyKerberos-1.3.0/pysrc/kerberos.py#L413